common/djangoapps/student/views/dashboard.py

```diff
@@ -629,6 +632,22 @@ def student_dashboard(request):
 
     enterprise_message = get_dashboard_consent_notification(request, user, course_enrollments)
 
+    recovery_email_message = None
+
+    # Update the tag info with 'enterprise_learner' if the user belongs to an enterprise customer.
+    secondary_email_exist = AccountRecovery.objects.filter(user=user).exists()
+    if is_secondary_email_feature_enabled_for_user(user=user) and not secondary_email_exist:
+        recovery_email_message = Text(
+            _(
+                "Add a recovery email to retain access when single-sign on is not available. "
+                "Go to {link_start}your Profile{link_end}.")
```

Comment:
Please change the text `your Profile` to `your Account Settings` as the secondary email (recovery email) address is present on the account settings page not on the profile page